_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -68,10 +68,13 @@ This rule is designed specifically for [2.4.4 Link Purpose (In Context)][sc244],
 
 This rule specifically targets links within the exact same context. Links with identical name that are in identical (but not the same) contexts also fail [2.4.4 Link Purpose (In Context)][sc244]. However, defining "identical context" unambiguously was not really possible and was left out of this rule.
 
+Elements that are [inheriting semantic][] `link` nodes within each test target are distinguishable each other by some user. This means that this rule applies to links with identical accessible names and context that are not [ambiguous to users in general](https://www.w3.org/TR/WCAG21/#dfn-ambiguous-to-users-in-general), which is the exception mentioned in Success Criterion 2.4.4 Link Purpose (In Context). If the links are ambiguous to users in general, users of assistive technologies are not at a disadvantage when viewing the links, which makes it more of a general user experience concern than an accessibility issue.
```

Comment:
I don't really understand the first sentence in this paragraph 🤔

Reply:
The target of this rule is a set of two or more elements that are [inheriting semantic][] `link` nodes.

These 2 or more element that are part of the target, that have the same acc name, same context, etc. are somehow distinguishable by some user, resulting in something that is not ambiguous to users in general.

I agree, it's a bit complex as written. 

Would be better to remove the first part, so:
```suggestion
This rule applies to links with identical accessible names and context that are not [ambiguous to users in general](https://www.w3.org/TR/WCAG21/#dfn-ambiguous-to-users-in-general), which is the exception mentioned in Success Criterion 2.4.4 Link Purpose (In Context). If the links are ambiguous to users in general, users of assistive technologies are not at a disadvantage when viewing the links, which makes it more of a general user experience concern than an accessibility issue.
```

Reply:
I prefer the revision you propose. However, I do not understand why this is in the Accessibility Support section instead of the Background section. This is not related to assistive technology limitations.

Reply:
Right, moving to background notes.